examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -0,0 +1,910 @@
+{
```

Comment:
<p>I believe since this is still based on HF architectures, we need to provide a max_length, and under the hood we still pad the sequences, even duh we do not pad it in the preprocessing. Can you confirm with Sara?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>reached out to Sara to confirm</p>

Reply:
<p>checked with Sara, based on the response I believe this is okay, here is the description of how all these pieces fit together in MMs:</p><p> </p><blockquote>"In Merlin models, we removed the max-length parameter and relies on the tf.ragged properties to automatically pad the sequences of each batch before applying the HF transformer layer.</blockquote><p>In fact, converting a ragged to a dense tensor will automatically pad the sequences of a given batch to the maximum length found in this batch. So the max length is also variable from one batch to another"</p>